postmark/django_backend.py

```diff
@@ -135,7 +137,7 @@ def _send(self, messages):
                 return False
         else:
             pm_messages = list(map(self._build_message, messages))
-            pm_messages = [m for m in pm_messages if m != False]
+            pm_messages = [m for m in pm_messages if not m]
```

Comment:
This looks like an incorrect substitution, shouldn't it be `if m`? Original condition is `if m is not false` where the change is now `if m is falsey`

Reply:
Oops. Good catch. @nicholasserra

Reply:
Looks like a spot where I need to add some test coverage :P